mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -1118,8 +1118,14 @@ static SafeSocketHandle Accept_internal (SafeSocketHandle safeHandle, out int er
 
 #region Bind
 
+#if FEATURE_NO_BSD_SOCKETS
+		[Obsolete  ("System.Net.Sockets.Socket:Bind is not supported on this platform.")]
+#endif
```

Comment:
I think we should add Obsolete attribute. On the exception message for PNSE, it's not very useful (you get stacktrack with the type) but I am ok with it